C.js

```diff
@@ -1,77 +1,266 @@
-let taskNumber = 1;

```

Comment:
this code is INSANLY long, almost 300 lines of vanilla js code! 🤯
separate this code to files

Reply:
do this in part C